src/shared/components/case-editor/domain/wizard-page-field.model.ts

```diff
@@ -1,7 +1,10 @@
 import { Orderable } from '../../../domain/order/orderable.model';
+import { ComplexFieldMask } from './wizard-page-field-complex-mask.model';
 
 export class WizardPageField implements Orderable {
   case_field_id: string;
   order?: number;
   page_column_no?: number;
+  display_context?: string;
+  complex_field_mask_list?: ComplexFieldMask[];
```

Comment:
`complex_field_mask_list` -> `complex_field_overrides` maybe better?

Reply:
Correct me if I'm wrong, but considering the UI only code, this override mechanism is quite generic right? I mean, it could be applied to any field type?
In that case, we could keep namings a bit more generic, like `fieldOverrides`?

Reply:
It will work only for fields marked with COMPLEX displayContext in CaseEventToFields Excel tab. I will change to _complex_field_overrides_